roxie/ccd/ccdserver.cpp

```diff
@@ -1400,14 +1421,50 @@ class CRoxieServerActivity : implements CInterfaceOf<IRoxieServerActivity>, impl
         }
 #endif
         executeDependencies(parentExtractSize, parentExtract, 0);
-        if (input)
-            input->start(parentExtractSize, parentExtract, paused);
         ensureCreated();
         basehelper.onStart(parentExtract, NULL);
+        startInputs(parentExtractSize, parentExtract, paused);
```

Comment:
There are situations where startInputs can be executed at the same time as executeDependencies.  However filters, ifs and filter-projects can't easily.  Something to revisit if it proves significant.